include/sqlpp11/logic.h

```diff
@@ -38,9 +38,22 @@ namespace sqlpp
     struct logic_helper;
 
     // see http://lists.boost.org/Archives/boost/2014/05/212946.php :-)
+	template <bool... B>
```

Comment:
Seriously? Is there an issue with all_t (or its usage) below? Wow. This is annoying.

Please add a reference to the corresponding issue at MS.